src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Components/SearchBox/BitSearchBoxDemo.razor

```diff
@@ -43,4 +43,15 @@
             <BitSearchBox Placeholder="Search" IsEnabled="false"></BitSearchBox>
         </ExamplePreview>
     </ComponentExampleBox>
+
+    <ComponentExampleBox Title="Validation SearchBox" HTMLSourceCode="@example6HTMLCode" CSharpSourceCode="@example6CSharpCode" ExampleId="example6">
+        <ExamplePreview>
+            <div class="example-desc">BitSearchBox can be validation value.</div>
```

Comment:
what does this title mean?
it seems it needs some grammar check @maryamhdr

Reply:
Yeah, I think the title and the description must be modified grammatically. Also, the provided content is not proper.